doc/guides/nest2_to_nest3/nest2_to_nest3_overview.rst

```diff
@@ -642,8 +642,8 @@ use ``nest.spatial.grid()`` or ``nest.spatial.free``.
                                                          num_dimensions=2))
     nest.PlotLayer(free_nodes);
 
-.. image:: ../_static/img/NEST3_24_0.png
-
+.. image:: ../../_static/img/NEST3_24_0.png
+  :width: 500
```

Comment:
Should this have a unit, such as px? Why is a width specified for this image but not others?

Reply:
> Should this have a unit, such as px?

@clinssen Thanks for spotting this. The image renders well with and without the px unit. Whichever option we vote for, we should ensure all images that specify ``:width:`` consistently use that style. 

> Why is a width specified for this image but not others?

I specified the width of these two grids for now, since one of them was very small and the other one much bigger previously:  
![conversion_guide_images_2](https://user-images.githubusercontent.com/58212853/79220156-cb838180-7e53-11ea-9a10-0a5f0e02c1bd.png)

I did not touch the other plots, as their original size is suitable. Do you think we should specify width it for all?

Reply:
I think that ideally, we should improve docutils so it auto-detects the width and height of a figure from file, but until that day we will have to specify it manually. Without :width: and :height:, the browser is left guessing about the \<img> width/height, which is not a good thing.

I'm not however suggesting to make that part of this PR, it would go beyond the scope. Perhaps you have a suitable todo list it can be added to. I would have a slight preference for adding px units, but this is maybe also one for the follow-up.

Reply:
@clinssen You are right about the browser aspect, and we can certainly follow up on the docutils improvement in a separate PR in the future. 

For now, I have added the px units to the ``:width:`` as you suggested.